aten/src/ATen/core/ivalue_inl.h

```diff
@@ -192,9 +193,21 @@ struct GenericDict;
 
 struct CAFFE2_API Tuple : public List<IValue> {
   using List<IValue>::List;
+
```

Comment:
btw, in your serialization PR you'll need to fix up the pickler as well—currently the `type` member will just get ignore and it'll get pickled as a tuple, which isn't what we want.